spec/lib/tasks/eu_exit_business_finder_spec.rb

```diff
@@ -0,0 +1,170 @@
+require 'rake'
+require 'rails_helper'
+
+RSpec.describe 'eu_exit_business_finder:update_subscriber_list_titles_and_facet_values' do
+  before :each do
+    Rails.application.load_tasks
+  end
+
+  def mock_facet_values_to_remove(values)
+    allow(EuExitFacetMigrationConfig).to receive(:facet_values_to_remove).and_return(values)
+  end
+
+  def mock_facet_values_to_replace(replacements)
+    allow(EuExitFacetMigrationConfig).to receive(:facet_values_to_replace).and_return(replacements)
+  end
+
+  def mock_facet_value_labels_to_replace(replacements)
+    allow(EuExitFacetMigrationConfig).to receive(:facet_value_labels_to_replace).and_return(replacements)
+  end
+
+  def stub_find_eu_exit_guidance_business_json
+    abs_path_to_fixture_data = File.join(File.dirname(__FILE__), './fixtures/find-eu-exit-guidance-business.json')
```

Comment:
Would it be possible to use `Rails.root` rather than `File.dirname(__FILE__)`?  I think it looks slightly nicer.